### ISSUE TITLE
Updates to monitors outside terraform breaks plan/apply

### ISSUE THREAD

**Issue by jdpaton** [terraform-providers/terraform-provider-datadog#31]
### Terraform Version
Terraform v0.9.11

### Affected Resource(s)
Please list the resources as a list, for example:
- datadog_monitor

### Expected Behavior
After a monitor was manually updated in the datadog web UI, its parameters should have been overwritten by any terraform apply.

### Actual Behavior
It failed at the plan stage on a null reference.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1.  Create a monitor with the terraform datadog provider
2. In the datadog web UI, edit the monitor to change something, in this case thresholds.
3. Try to run the terraform plan again and it will give an error like:

```Error refreshing state: 2 error(s) occurred:

* module.monitoring.datadog_monitor.frontend_error_rate_percent: 1 error(s) occurred:

* module.monitoring.datadog_monitor.frontend_error_rate_percent: datadog_monitor.frontend_error_rate_percent: strconv.ParseInt: parsing "null": invalid syntax
* module.monitoring.datadog_monitor.frontend_response_time_2xx_p99: 1 error(s) occurred:

* module.monitoring.datadog_monitor.frontend_response_time_2xx_p99: datadog_monitor.frontend_response_time_2xx_p99: strconv.ParseInt: parsing "null": invalid syntax
```

**Comment by dtan4** [comment]
I got the same error... this patch https://github.com/zorkian/go-datadog-api/pull/129 may fix this problem.

**Comment by philwhln** [comment]
I'm hitting an issue with this to. If `Do not notify if data is missing` is configured then `no_data_timeframe` is `null` and it seems that terraform is expecting an `Int` here and so croaks.
```
{
	"options": {
		"no_data_timeframe": null,
```
```
strconv.ParseInt: parsing "null": invalid syntax
```
This happens on `terraform import` or `terraform plan`.

btw, hi @jdpaton! Hope things are going well :)

**Comment by nagas** [comment]
+1

**Comment by ojongerius** [comment]
Heads up; I've just merged https://github.com/zorkian/go-datadog-api/pull/129.

**Comment by ojongerius** [comment]
FYI: Merge has been included in release https://github.com/zorkian/go-datadog-api/releases/tag/v2.6

**Comment by catsby** [comment]
Fixed in #34 , I'll cut release 1.0.1 of this provider shortly

**Comment by catsby** [comment]
`v1.0.1` version is out: https://releases.hashicorp.com/terraform-provider-datadog/1.0.1/

Running `terraform init` should pull down the new version, unless you have it pinned
